syntax = "proto3";

package smn_dict;

enum EDict{
	None = 0;
	rip_rpc_itf_Login_Test1_Ret = 1;
	rip_rpc_itf_Login_Test2_Prm = 2;
	rip_rpc_itf_Login_Test2_Ret = 3;
	rip_rpc_itf_Login_DoLogin_Prm = 4;
	rip_rpc_itf_Login_DoLogin_Ret = 5;
	rip_rpc_itf_Login_Test1_Prm = 6;
	smn_base_Call = 7;
	smn_base_Ret = 8;
	smn_base_FPkg = 9;
}
